src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +43,10 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+
+        return numbers.stream()
+            .filter(w -> w > 3)
+            .map(w -> w * 2)
+            .reduce(0, (x, y) -> x + y);
```

Comment:
이렇게 `초기수치 0, 그후 x와 y를 더해간다`라는 식으로 reduce를 정의해도 되지만 다음과 같은 방식도 있을것 같습니다.
```
.reduce(Integer::sum).get()
```
결과는 같지만, 어떤방식이 더 좋은방식일까 고민됩니다.
개인적으로는 지금 구현한 방식이 함수의 연산이 그대로 노출되므로 읽기쉽고, 변경도 편하지 않을까 생각하고 있습니다. 어떻게 생각하시나요?

Reply:
넵! 코멘트 주신 부분처럼 제 생각도 Integer::sum사용하셔도 될 것 같아요! 
아래 처럼 생각하신 부분 맞으실까용?

```suggestion
 return numbers.stream()
            .filter(w -> w > 3)
            .map(w -> w * 2)
            .reduce(0, Integer::sum);
```

Reply:
코멘트 감사합니다.
네. get()을 취하는형태가 아닌것 빼고는 생각하는대로 인것 같습니다.
알려주셔서 감사합니다!